Gemfile.lock

```diff
@@ -104,10 +107,15 @@ GEM
     jbuilder (2.11.5)
       actionview (>= 5.0.0)
       activesupport (>= 5.0.0)
+ feature6/views
+    loofah (2.21.2)
+      crass (~> 1.0.2)
+      nokogiri (>= 1.12.0)
```

Comment:
- [ ] Please your User Controller `#show` action should render the user params[:id] as shown below.
```ruby
# app/controllers/users_controller.rb

def show
  @user = User.find(params[:id])
end

```

- [ ] Please you `app/views/users/show.html.erb` file should use the `#recent_post` method to render the user's 3 most recent post.
- [ ] Please the **See all Post** links opens user's post index as displayed on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png)

Reply:
- Modify the show action as suggested here. It's a good idea to make changes before submitting the PR for review.